src/app/shared/service/users.service.ts

```diff
@@ -50,7 +50,9 @@ export class UsersService {
       }
     }
     for (const id in musicFolderId) {
-      params = params.set("musicFolderId", id);
+      if (musicFolderId.hasOwnProperty(id)) {
```

Comment:
But it anyway not iterate over musicFolderId if it is empty right ?

Reply:
I would have thought that to but the IDE was complaining about it so thought I'd fix it.  Haven't got much time to look at the code but is musicFolderId just a single ID? If so is the loop even needed?

Reply:
Yes, it is needed.
From the subsonic API:
```
musicFolderId | No | All folders | (Since 1.12.0) IDs of the music folders the user is allowed access to. Include the parameter once for each folder.
```